### ISSUE TITLE
 Only InputStream, Json and void are supported as return types for route methods

### ISSUE THREAD

**Issue by timotheecour** [rejectedsoftware/vibe.d#1467]
couldn't vibe simply try to call serializeToJson on arguments that are not Json/void ?

**Comment by s-ludwig** [comment]
Yes, the choice was conservative to avoid running into any dead ends later on, but together with support for custom serialization (e.g. XML or a different JSON document structure) this should definitely be added.
